docs/manual-CN/5.appendix/cypher-ngql.md

```diff
@@ -0,0 +1,155 @@
+# Comparison Between Cypher and nGQL
+
+
+## 基本概念对比
+
+|Name               | Cypher | nGQL          |
+| --- | --- | --- |
```

Comment:
Shall we change the first column to Chinese?

Reply:
yep